ghost/core/core/server/web/parent/middleware/queue-request.js

```diff
@@ -0,0 +1,76 @@
+const debug = require('@tryghost/debug')('middleware:queue-request');
+const metrics = require('@tryghost/metrics');
+const path = require('node:path');
+const expressQueue = require('express-queue');
+
+const DEFAULT_QUEUE_LIMIT = 100;
```

Comment:
All our testing was with this at 20 and it seemed throughput was reasonably fine. Do we have data to know that 100 is ok? Even locally we can see ghost fall over with ~70rps.

Reply:
Definitely not 😄 We have a task to workout what this specific value should be, this was just a placeholder till we worked that out